src/content/docs/browser/new-relic-browser/configuration/experimental-features.mdx

```diff
@@ -4,7 +4,7 @@ metaDescription: "Opt-in to use experimental features in New Relic browser monit
 freshnessValidatedDate: never
 ---
 
-New Relic Browser features are exposed to customers in a controlled manner to ensure stability and reliability. However, some features are made available before they reach general availability. These are known as experimental features. To access them, you must opt in.
+New Relic Browser features are exposed to customers in a controlled manner to ensure stability and reliability. However, some features are made available before they reach general availability. These are known as experimental features. To gain access to features before they are generally available, you must opt in.
```

Comment:
New Relic Browser features are released gradually to ensure stability and reliability. However, you can opt in to avail yourself of some features before their GA. These are known as experimental features.